apps/ledger-live-desktop/src/renderer/components/ErrorBody.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { BoxedIcon, Text } from "@ledgerhq/react-ui";
+import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
+import styled from "styled-components";
+
+export const ErrorTitle = styled(Text).attrs({
+  color: "neutral.c100",
+  textAlign: "center",
+})``;
```

Comment:
[ASK]: what is the benefit to use `attrs` to edit style ?

you can also do something like

```ts
export const ErrorTitle = styled(Text)`
color: ${p => p.theme.colors.neutral.c100};
text-align: center;
`
```

Reply:
yes I think you're right, let me change this